ui/pages/remote-mode/hooks/useRemoteMode.ts

```diff
@@ -166,9 +168,19 @@ export const useRemoteMode = ({ account }: { account: Hex }) => {
     [account, delegationManagerAddress, globalNetworkClientId],
   );
 
+  const listDelegations = useCallback(async (filter: DelegationFilter) => {
+    return await listDelegationEntries(filter);
+  }, []);
+
+  const getDelegation = useCallback(async (hash: Hex) => {
+    return await getDelegationEntry(hash);
+  }, []);
+
```

Comment:
This shouldn't be done this way. We actually need to remove these actions (I added a TODO in `/selectors/delegation.ts`)

What you should do instead:
- for list, just `const delegationEntries = useSelector(listDelegationEntries)` (this `listDelegationEntries` comes from `/selectors/delegation.ts`)
- do the same for `getDelegationEntry`, but you'll have to implement the selector in `/selectors/delegation.ts`

The reason to do this and not call the controller directly is so that the UI is reactive to the controller's state changes. That is why it might not even make sense for us to have methods in the controller to do read operations (since we should just tap into its state).

Reply:
Good call! Just added an initial pass at switching over to use the selector (47e2cac567c77b749d389dc696c03f12d30a04cb)